cli/Cargo.toml

```diff
@@ -41,7 +41,7 @@ solana-cli-config = "1.16"
 solana-faucet = "1.16"
 solana-program = "1.16"
 solana-sdk = "1.16"
-solang-parser = "=0.3.2"
+solang-parser = "=0.3.3"
```

Comment:
Just to confirm: is it necessary for us to use a fixed version(`=`) of `solang-parser`?

Reply:
The `solang-parser` crate produces a parse tree, and when solidity introduces new language features, we have to change this data structure. This is usually impossible to do in a backwards compatible way, so `solang-parser` can break compatibility between versions. 
We don't want the anchor build to be broken because there is a new version of `solang-parser` which has a slightly different parse tree.

Reply:
We have a comment in our `README.md`: https://github.com/hyperledger/solang/blob/main/solang-parser/README.md?plain=1#L8
This could probably do with a comment in `Cargo.toml`.

Reply:
Got it, thanks for the explanation.